### DISCUSSION TITLE
Dark reader UI changes

### DISCUSSION THREAD

**Discussion by imadzanjani** [darkreader/darkreader#7185] [Q&A] [answered]
A few months ago the control center had a different more rounded and modern UI. But it has since gone back to the old boxy UI again. Does anyone know why and how, if, I can go back to the rounded version?

**Comment by Gusted** [comment, marked as answer]
It was a recent change to update the UI to be a more polished version, it's still a preview design so the design is not final yet. It's not possible to go back to the rounded version.

**Reply by imadzanjani** [reply]
Aww that's sad. Any idea when its gonna come out? @Gusted

**Comment by alexanderby** [comment]
@DragonEmperor19 I will finish with some bureaucracy and will continue working on the design.

**Reply by imadzanjani** [reply]
Cool! Can't wait for it!

**Comment by redudant-overflowing** [comment]
Hey, I made a mockup if you are interesting in gathering some ideas! o/

https://github.com/darkreader/darkreader/discussions/7494
